cypress/integration/OrganizationContainerMissingGridID.test.ts

```diff
@@ -0,0 +1,19 @@
+describe('OrganizationContainer Display GridID if it exists', () => {
+  beforeEach(() => {
+    cy.setCookie('_consent', 'true')
+  })
+  it('visit site with gridID, id element should exist', () => {
+    cy.visit('/ror.org/052gg0110')
```

Comment:
Is there a difference between the URL here, starting with a slash, and the one below which doesn't?

Reply:
I'm not aware of one and it wasn't intentional.  Even though it passed, I will update to be consistent.